status-im/status-react
# Remove externs and add them to gitignore

## Summary:
We are using simple optimizations for prod builds so this file is not used.

We have much more useful output with `:simple` optimizations if an error
happens, not just something like `a.b.adsfas is undefined`.

The difference in size between advanced and simple optimizations is only
about 300KB, and the RN packager also makes some magic with js after cljs
compiler anyway.

### Steps to test:
- remove node_modules and reset npm cache
```sh
rm -rf node_modules
npm install
npm --reset-cache start
```

- react-native run-android should still work
```sh
react-native run-android
```

status: WIP
